script/legacy/legacy30.php

```diff
@@ -102,16 +102,22 @@ function isKeyboardFiltered($keyboard_id) {
   function getLanguages($id, $keyboardid) {
     global $device;
 
+    $languages = DB_LoadLanguages_0($id);
+    $keyboards_0 = DB_LoadLanguages_0_Keyboards($id);
+    $keyboards = [];
+    foreach($keyboards_0 as $row) {
+      $keyboards[$row['keyboard_id']] = $row;
+    }
```

Comment:
My IDE linter said `$reskbds` was undefined in line 125.
legacy40.php sets it null right before `$LastID = '';`.